test/integration/targets/postgresql/tasks/pg_hba.yml

```diff
@@ -0,0 +1,35 @@
+- name: Make sure file does not exist
+  file:
+    dest=/tmp/pg_hba.conf
+    state=absent
+- name: Add several ip addresses
```

Comment:
`check_mode` (meaning: file isn't created/updated when check mode is enabled) and idempotence (meaning: changed is `False` when a task is repeated) should be checked.